docuware-connector/processes/UploadService.p.json

```diff
@@ -71,7 +71,9 @@
           "  in.document = DocuWareService.uploadFile(client, in.uploadFile, in.configuration, in.propertiesList );",
           "}catch(DocuWareException dex){",
           "  in.error = BpmError.create(\"com:eon:ivy:shared:docuware:rest\").withMessage(dex.getErrorMessage()).build();",
-          "  ivy.log.info(\"Document upload failed. Error: \" + dex.getErrorMessage());",
+          "  ivy.log.error(\"Document upload failed. Error: \" + dex.getErrorMessage());",
+          "} catch(Exception e){",
```

Comment:
please format the try catch here:
"try{",
          "  in.document = DocuWareService.uploadFile(client, in.uploadFile, in.configuration, in.propertiesList );",
          "} catch(DocuWareException dex) {",
          "  in.error = BpmError.create(\"com:eon:ivy:shared:docuware:rest\").withMessage(dex.getErrorMessage()).build();",
          "  ivy.log.error(\"Document upload failed. Error: \" + dex.getErrorMessage());",
          "} catch(Exception e) {",